src/views/Tree.vue

```diff
@@ -53,15 +53,21 @@ const QUERIES = {
           id
           name
           status
+          owner
+          host
+          port
           taskProxies {
             id
             state
             cyclePoint
             task {
+              meanElapsedTime
```

Comment:
The Tree View isn't actually using this data yet though...

Reply:
That's correct, I separated the pull requests. I thought this would be reviewed much quicker, and then either the Tree View PR could be updated, or it could be done in another PR.

Reply:
OK, probably easier to merge PRs for the future so the subscription can't get out of date with the component?

Reply:
(and sorry it's been hanging around for so long)